File: frontend/metropolitan/src/app/shared/model/sucursales.ts
export interface Sucursal {
  id?: number;
  nombre?: string;
  direccion?: string;
  telefonos: string;
  email: string;
  idEncargado: number;
  createBy?: number;
  modify?: number;
  createDate?: string;
  modifyDate?: string;
}

File: backend/Metropolitan/Common/model/sucursales.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataBase.model
{
    public class sucursales
    {
        public int id;
        public string nombre;
        public string direccion;
        public string telefono;
        public string email;
        public int idEncargado;
        public int createBy;
        public DateTime createDate;
        public int modify;
        public DateTime modifyDate;
    }
}
